source/slang/slang.cpp

```diff
@@ -1348,6 +1347,59 @@ SLANG_NO_THROW slang::TypeReflection* SLANG_MCALL Linkage::specializeType(
     return asExternal(specializedType);
 }
 
+DeclRef<GenericDecl> getGenericParentDeclRef(
```

Comment:
can just use `getInnermostGenericParent`?

Reply:
This is a bit different unfortunately. 
This function returns the decl-ref of the generic-decl while `getInnermostGenericParent` returns the decl-ref of the inner decl of the closest generic-decl (this is so that substitutions are available)